engine/table/src/main/java/io/deephaven/engine/table/impl/sources/regioned/RegionedPageStore.java

```diff
@@ -133,11 +143,12 @@ private static long validateMask(final long mask, final String name) {
     /**
      * A regioned page store for use when the full set of regions and their sizes are known.
      */
-    abstract class Static<ATTR extends Any, INNER_ATTR extends ATTR, REGION_TYPE extends Page<INNER_ATTR>>
+    abstract class Static<ATTR extends Any, INNER_ATTR extends ATTR, REGION_TYPE extends ColumnRegion<INNER_ATTR>>
```

Comment:
If the bound is Page instead of ColumnRegion, we can't interrogate the supportsUnboundedFill method.